docs/concepts/writing-config.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Writing Configuration
+headline: Writing Configuration
+sidenav: doc-side-concepts-nav.html
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+{% capture overview %}
+This document describes how to write configuration that conforms to Istio's schemas. All configuration schemas in Istio are defined as [protobuf messages](https://developers.google.com/protocol-buffers/docs/proto3). When in doubt, search for the protos.
+{% endcapture %}
+
+{% capture body %}
+
+## Translating to YAML
+There is no canonical mapping between protobufs and YAML; instead [protobuf defines a canonical mapping to JSON](https://developers.google.com/protocol-buffers/docs/proto3#json), and [YAML defines a canonical mapping to JSON](http://yaml.org/spec/1.2/spec.html#id2759572). To ingest YAML as a proto we convert it to JSON then to  protobuf.
+
+**Important things to note:**
+- YAML fields are implicitly strings
+- Proto `repeated` fields map to YAML lists; each element in a YAML list is prefixed by a dash (`-`)
+- Proto `message`s map to JSON objects; in YAML objects are field names all at the same indentation level
+- YAML is whitespace sensitive and must use spaces; tabs are never allowed
+
+### Proto `map` and `message` fields:
```

Comment:
Can you change these headers? They don't look good in the right-hand side table of contents. I think you can generally remove the word "Proto" on each of these, and drop the : at the end of the line.